app/sources/colombia.py

```diff
@@ -55,8 +55,9 @@ def _get_remote_contract(
 ) -> tuple[list[dict[str, str]], str]:
     params = f"proceso_de_compra='{proceso_de_compra}' AND documento_proveedor='{proveedor_adjudicado}'"
     if previous:
-        contract_url = f"{contract_url} AND fecha_de_firma IS NOT NULL"
-    return util.loads(sources.make_request_with_retry(contract_url, HEADERS)), contract_url
+        params = f"{params} AND fecha_de_firma IS NOT NULL"
+    contract_url = f"{URLS['CONTRACTS']}?$where={quote_plus(params)}"
+    return sources.make_request_with_retry(contract_url, HEADERS).json(), contract_url
```

Comment:
Need to keep util.loads from merge conflict

Reply:
Done